openedx/features/course_experience/tests/views/test_course_home.py

```diff
@@ -71,22 +69,13 @@ def tearDown(self):
         remove_course_updates(self.course)
         super(TestCourseHomePage, self).tearDown()
 
-    @override_flag(UNIFIED_COURSE_EXPERIENCE_FLAG, active=True)
-    def test_unified_page(self):
-        """
-        Verify the rendering of the unified page.
-        """
-        url = course_home_url(self.course)
-        response = self.client.get(url)
-        self.assertContains(response, '<h2 class="hd hd-3 page-title">Test Course</h2>')
```

Comment:
@andy-armstrong: I killed this test because with both a True or False flag, the page has the same title and this passes.  I figure the welcome message tests are good enough.  Or, if you want a new test, we can discuss.